Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -126,7 +127,10 @@ const DefineConversationV2: React.FC<DefineConversationProps> = (props) => {
   const normalizeTemplateId = () => {
     if (currentTemplate) {
       // use almost the same patterns as in assetManager.ts
-      return currentTemplate.name.trim().replace(/[-\s]/g, '_').toLocaleLowerCase();
+      const camelCasedName = camelCase(
+        currentTemplate.name.trim().toLocaleLowerCase().replace('bot', '').replace('maker', '').replace(/-/g, ' ')
```

Comment:
Does it make more sense to use case insensitive regex in the replace calls instead of making the name lower case? Might be able to combine the regex for 'bot' and 'maker' too since they're both  
```suggestion
        currentTemplate.name.trim().replaceAll(/bot|maker/gi, '').replace(/-/g, ' ')
```

Reply:
Yeah thats a bit cleaner, I have made the change